src/main.ts

```diff
@@ -1,11 +1,13 @@
 import express, { Request, Response, NextFunction} from 'express';
 import userRoutes  from "./routes/users";
+import politicianRoutes  from "./routes/politicians";
 
 
 const app = express();
 
 
 app.use("/users", userRoutes);
+app.use("/politicians", politicianRoutes);
 app.use(express.json());

```

Comment:
`app.use(express.json())`
は、これを呼び出した以降req.bodyでjsonを構造体として読めるようになるものなので、
`app.use("/users", userRoutes);`
の上に書かないと正常に動かないです
```suggestion
app.use(express.json());
app.use("/users", userRoutes);
app.use("/politicians", politicianRoutes);
```